### PR TITLE
rspec tests for group controller

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/spec/controllers/webui/groups_controller_spec.rb [hunk 1]
```diff
@@ -64,4 +65,79 @@
...
+    context 'as a group maintainer' do
+      before do
+        create(:group_maintainer, user: user, group: group)
+      end
+
+      it 'shows edit form and populates it with data' do
+        get :edit, params: { title: group.title }
+
+        expect(response).to have_http_status(:success)
+        assigned_members = assigns(:members).map(&:values).flatten
```
Comment (Ana06) [inline src/api/spec/controllers/webui/groups_controller_spec.rb]: what about writing it like:

```` Ruby
assigns(:members).map{ |user| user['name'] }

File: src/api/spec/controllers/webui/groups_controller_spec.rb [hunk 2]
```diff
@@ -64,4 +65,79 @@
...
+    end
+
+    context 'as an admin' do
+      before do
+        login(create(:admin_user))
+      end
+
+      context 'creating a new group' do
+        it 'creates a group with members' do
+          post :create, params: { group: { title: 'my_group', members: users_to_add.map(&:login).join(',')}}
```
Comment (Ana06) [inline src/api/spec/controllers/webui/groups_controller_spec.rb]: why here https://github.com/openSUSE/open-build-service/pull/2823/files#diff-587a38e2b32b8b815600f0c307b62ebeR112 you don't need the join and here you need it if it is the same method? 😕

Comment (bgeuken) [inline src/api/spec/controllers/webui/groups_controller_spec.rb]: Just forgot to add it there as well. The reason why it doesn't cause a failure here is that the permission check runs first.

Comment (Ana06) [inline src/api/spec/controllers/webui/groups_controller_spec.rb]: :+1:

**Pull request merged**